enlighten/measurement/Measurement.py

```diff
@@ -318,6 +315,11 @@ def __init__(self,
             self.timestamp          = datetime.datetime.now()
             self.baseline_correction_algo = spec.app_state.baseline_correction_algo
```

Comment:
Technically we could have populated plugin_name here, as long as one was connected and enabled.